__tests__/app.test.js

```diff
@@ -37,3 +38,43 @@ describe("/api/topics", () => {
       });
   });
 });
+
+describe("/api/articles", () => {
+  it("200: should respond with an array of article objects with the following properties: author, title, article_id, topic, created_at, votes, comment_count, sorted by created date in decending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Array);
+        body.forEach((articles) => {
+          expect.objectContaining({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
```

Comment:
we **don't** want to include our article body in the response for all articles, only when we get to the articles by id endpoint. Just need to be more specific in the columns you are selecting in your SQL statement